app/Console/Commands/AddExpression.php

```diff
@@ -99,17 +104,31 @@ public function handle()
                     }
                 }
                 $expressions = file_exists(base_path($path)) ? require base_path($path) : [];
-                if (isset($expressions[$expression_key])) {
-                    if (! $this->confirm('Do you want to override the old '.($language == 'en' ? 'english' : 'hungarian').' translation ('.$expressions[$expression_key].')?')) {
-                        $this->info('Action cancelled.');
+                if ($file == 'validation') {
+                    if (isset($expressions['attributes'][$expression_key])) {
+                        if (! $this->confirm('Do you want to override the old '.($language == 'en' ? 'english' : 'hungarian').' translation ('.$expressions['attributes'][$expression_key].')?')) {
+                            $this->info('Action cancelled.');
 
-                        return;
+                            return;
+                        }
+                    }
+                } else {
+                    if (isset($expressions[$expression_key])) {
+                        if (! $this->confirm('Do you want to override the old '.($language == 'en' ? 'english' : 'hungarian').' translation ('.$expressions[$expression_key].')?')) {
+                            $this->info('Action cancelled.');
+
+                            return;
```

Comment:
For later: we might not want to return here, if we only wish to update the translation for the other language.